src/ol/interaction/selectinteraction.js

```diff
@@ -264,7 +264,7 @@ ol.interaction.Select.handleEvent = function(mapBrowserEvent) {
          * @param {ol.layer.Layer} layer Layer.
          */
         function(feature, layer) {
-          var index = goog.array.indexOf(features.getArray(), feature);
+          var index = features.getArray().indexOf(feature);
           if (index == -1) {
```

Comment:
Or if #4181 makes it in first, this could be shortened to `if (!ol.array.includes(features.getArray(), feature) {}`.